//
// Academic License - for use in teaching, academic research, and meeting
// course requirements at degree granting institutions only.  Not for
// government, commercial, or other organizational use.
//
// ac_O_V.h
//
// Code generation for function 'ac_O_V'
//

#ifndef AC_O_V_H
#define AC_O_V_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void ac_O_V(double V_init, double A_init, double V_wayp, double V_max,
            double J_max, double J_min, creal_T t[14]);

#endif
// End of code generation (ac_O_V.h)
